libs/langgraph/src/tests/pregel.test.ts

```diff
@@ -9708,47 +9708,6 @@ graph TD;
     expect(thirdState.tasks).toHaveLength(0);
   });
 
-  it("should cancel when signal is aborted", async () => {
```

Comment:
Moved to new file because this one is massive and I'm actively splitting it up in another branch.